cdap-ui/app/cdap/components/PipelineConfigurations/Store/index.js

```diff
@@ -361,7 +366,10 @@ const configure = (state = DEFAULT_CONFIGURE_OPTIONS, action = defaultAction) =>
       let newCustomConfigs = {};
       Object.keys(action.payload.customConfig).forEach((newCustomConfigKey) => {
         let newCustomConfigValue = action.payload.customConfig[newCustomConfigKey];
-        if (action.payload.isBatch && state.engine === 'mapreduce') {
+        if (
+          GLOBALS.etlBatchPipelines.indexOf(action.payload.pipelineType) !== -1 &&
+          state.engine === 'mapreduce'
```

Comment:
use the ENGINE_OPTIONS enum for mapreduce?

Reply:
Fixed